### PR TITLE
shanoir-issue#1502-3: remove unused dependencies

### PR DESCRIPTION
**Pull request by pierrehenri-dauvergne**

### CODE REVIEW

File: shanoir-ng-front/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -432,10 +431,6 @@ let routes: Routes = [
         path: 'user',
         redirectTo: 'user/list',
     },
-    {
```
Comment (julien-louis) [inline shanoir-ng-front/src/app/app-routing.module.ts:435]: Why did you remove the user list from Shanoir ?

Comment (julien-louis) [reply shanoir-ng-front/src/app/app-routing.module.ts:435]: I'm pretty sure it's the user list you get at https://shanoir-ng-nginx/shanoir-ng/user/list . You have to be an administrator though.

Comment (pierrehenri-dauvergne) [reply shanoir-ng-front/src/app/app-routing.module.ts:435]: Yes indeed, you're right. This component was defined as unused by depcheck. I knew used dependencies could be listed by depcheck but I didn't pay enough attention here.
